lib/sync/fetcher.go

```diff
@@ -86,18 +96,12 @@ func (f *BlockFetcher) fetch(ctx context.Context, si *SyncInfo) error {
 	f.logger.Debug("Fetch start", "height", height)
 
 	n := f.pickRandomNode()
-	f.logger.Info("Fetching items from node", "node", n, "height", height)
+	f.logger.Info(fmt.Sprintf("Fetching items from node: %s", n), "fetching_node", n, "height", height)
```

Comment:
Isn't `n` duplicated?

Reply:
Yep, but msg has node is easy to see log :-)